guides/user/plugins/user-guide.md

```diff
@@ -1,109 +1,83 @@
 ---
 layout: single
-title:  "Users Guide"
+title:  "Plugin Users Guide"
 sidebar:
   nav: guides
 redirect_from:
   - /guides/user/plugin-users/
 ---
 
-{% include alpha version="1.19.4" %}
+{% include alpha version="1.20.6" %}
 
->Note: Spinnaker 1.19.x only supports backend plugins due to a bug in Deck.
+_Note: Spinnaker 1.19.x does not support frontend plugins due to a bug in Deck._
 
 {% include toc %}
 
-In this guide, you add an existing plugin from an [example repository](https://github.com/spinnaker-plugin-examples/examplePluginRepository) to Spinnaker. See the [Plugin Creators Guide](/guides/developer/plugin-creators/overview/) for how to create a plugin.
+## Overview
 
-# Requirements
+Spinnaker uses [PF4J-Update](https://github.com/pf4j/pf4j-update) to load and manage third-party plugins. These plugins can implement a PF4J extension point or be Spring components. See the [Plugin Creators Guide](/guides/developer/plugin-creators/overview/) for details.
```

Comment:
I might drop "third-party" here. All Spinnaker plugins use `pf4jUpdate` regardless of their origin.